lib/run-in-terminal.coffee

```diff
@@ -196,7 +199,8 @@ class ArgumentsRequester
 
         })
 
-        @line_edit_view.addEventListener("focusout", @save_and_hide)
+        # Request args does not work if below line is uncommented? (OS X)
+        # @line_edit_view.addEventListener("focusout", @save_and_hide)
```

Comment:
Yes, that is because `"focusout"` false fires. This works fine before some atom version. AFAIR, with this commented you can't hide line edit popup.